### PR TITLE
New entry point to get the endpoints of a segment.

### PR DESCRIPTION
**Pull request by pleroy**

### CODE REVIEW

File: ksp_plugin/plugin.hpp [hunk 1]
```diff
@@ -260,6 +260,11 @@ class Plugin {
       not_null<std::unique_ptr<NavigationFrame>> plotting_frame);
   virtual not_null<NavigationFrame const*> GetPlottingFrame() const;
 
+  virtual Position<World> BarycentricToPlottingFrame(
```
Comment (eggrobin) [inline ksp_plugin/plugin.hpp]: PlotBarycentricPosition, no plotting_frame is visible to the caller here.

File: ksp_plugin/plugin.cpp [hunk 2]
```diff
@@ -470,6 +470,25 @@ not_null<NavigationFrame const*> Plugin::GetPlottingFrame() const {
...
+Position<World> Plugin::BarycentricToPlottingFrame(
+    Instant const& t,
+    Position<Barycentric> const& position,
+    Position<World> const& sun_world_position) const {
+  auto const barycentric_to_world =
+      AffineMap<Barycentric, World, Length, OrthogonalMap>(
+          sun_->current_position(current_time_),
+          sun_world_position,
+          OrthogonalMap<WorldSun, World>::Identity() * BarycentricToWorldSun());
+  auto from_barycentric_to_navigation_at_t =
```
Comment (eggrobin) [inline ksp_plugin/plugin.cpp]: const, get rid of the froms.

File: ksp_plugin/plugin.cpp [hunk 3]
```diff
@@ -470,6 +470,25 @@ not_null<NavigationFrame const*> Plugin::GetPlottingFrame() const {
...
+      plotting_frame_->ToThisFrameAtTime(t).rigid_transformation();
+  auto from_navigation_frame_to_world_at_current_time =
```
Comment (eggrobin) [inline ksp_plugin/plugin.cpp]: same same

File: ksp_plugin/plugin.hpp (hunk 1 above)
Comment (pleroy) [inline ksp_plugin/plugin.hpp]: Done.

File: ksp_plugin/plugin.cpp (hunk 2 above)
Comment (pleroy) [inline ksp_plugin/plugin.cpp]: Done.

File: ksp_plugin/plugin.cpp (hunk 3 above)
Comment (pleroy) [inline ksp_plugin/plugin.cpp]: Done.

**Pull request merged**